### DISCUSSION TITLE
How to change the color of the header where the table name is written

### DISCUSSION THREAD

**Discussion by makarasty** [schemacrawler/SchemaCrawler#1450] [Q&A] [answered]
It's light purple now, and it's freaking me out
![image](https://github.com/schemacrawler/SchemaCrawler/assets/71918286/bf130ce0-bd3c-4abf-9707-19bc05a8ac25)

I've been trying to use:
- `sschemacrawler.graph.graphviz.node.header.fillcolor=black`,
- `schemacrawler.format.color_map.schema_name.tabletitle=black` but that, as you can see, didn't work.
(just speculation)

It would be good to change the color of the text in the header, it is changed using `schemacrawler.graph.graphviz.node.fontcolor=white`, but with the other text, and I want to separately

Сhatgpt4 offered a bunch of options, not one of which did not give a result, the most correct was to address here

I'm not quite sure how to work with .dot

**Comment by sualeh** [comment]
@makarasty You can specify colors for each schema in the "[schemacrawler.colormap.properties](https://github.com/schemacrawler/SchemaCrawler/blob/main/schemacrawler-docs/config/schemacrawler.colormap.properties)" file. This will be in the same place as the other properties file you were editing.

**Reply by makarasty** [reply]
I still don't understand what I need to write in `schemacrawler.colormap.properties` to change the color, also how to bind it to the starter?

**Comment by sualeh** [comment, marked as answer]
@makarasty Edit "[schemacrawler.colormap.properties](https://github.com/schemacrawler/SchemaCrawler/blob/main/schemacrawler-docs/config/schemacrawler.colormap.properties)" file and add a line like this (caveat: I have not tested this):

```
FFFF00=["`]?fix\.ua["`]?
```

You will find a "schemacrawler.colormap.properties" already present in your distribution.

**Reply by makarasty** [reply]
cool, this is the answer to my question in this discussion. I want to know how to change the text color only for header fix.ua....
Or what to read to understand how to work colormap.properties
